iolog.c

```diff
@@ -97,13 +97,13 @@ static void iolog_delay(struct thread_data *td, unsigned long delay)
 	delay -= usec;
 
 	fio_gettime(&ts, NULL);
-	while (delay && !td->terminate) {
-		this_delay = delay;
-		if (this_delay > 500000)
-			this_delay = 500000;
 
-		usec_sleep(td, this_delay);
-		delay -= this_delay;
+	while(delay && !td->terminate) {
```

Comment:
Style: missing space between ```while``` and parens start.

Reply:
Thank you for the catch!  I have fixed it in the new commit.

Reply:
I don't see the new commit?